pkg/api/controller_test.go

```diff
@@ -1896,7 +2026,7 @@ func TestParallelRequests(t *testing.T) {
 	config.HTTP.Port = SecurePort1
 	htpasswdPath := makeHtpasswdFileFromString(getCredString(username, passphrase))
 
-	// defer os.Remove(htpasswdPath)
+	//defer os.Remove(htpasswdPath)
```

Comment:
several commented-out defers here?

Reply:
Yes, because some tests are running in separate go routine and defer invokes before their completion

Reply:
That would explain *removing* them.

Reply:
Yes I forgot to remove, removed now.